modin/pandas/test/dataframe/test_indexing.py

```diff
@@ -2047,6 +2058,44 @@ def _make_copy(df1, df2):
         eval_setitem(modin_df, pandas_df, df_value[["value_col1"]], key)
 
 
+def test_setitem_2d_insertion():
+    md_df, pd_df = create_test_dfs(test_data["int_data"])
+
+    # Easy case - key and value.columns are equal
+    md_value, pd_value = create_test_dfs(
+        {"new_value1": np.arange(len(md_df)), "new_value2": np.arange(len(pd_df))}
```

Comment:
Why use `len(md_df)` in one place and `len(pd_df)` in other? Looks a bit confusing.

Reply:
done, now both use `len(md_df)`